src/plugins/uk.ac.kcl.mdeoptimise.ui/plugin.xml

```diff
@@ -434,12 +434,12 @@
 		</launchConfigurationTabGroup>
 	</extension>
 	<extension point="org.eclipse.jdt.core.classpathContainerInitializer">
-		<classpathContainerInitializer id="MDEOptimiser" class="uk.ac.kcl.ui.classpath.MDEOClassPathContainerInitializer" />
+		<classpathContainerInitializer id="uk.ac.kcl.mde_optimise.MDEO_CONTAINER" class="uk.ac.kcl.ui.classpath.MDEOClassPathContainerInitializer" />
```

Comment:
I'm not much bothered by this, but is there a particular reason why you chose this ID?

Reply:
A very poor attempt at consistency. Actually we have several namespaces through this project and they should all be changed to one.

We have:

uk.ac.kcl.ui
uk.ac.kcl.mdeoptimise.
uk.ac.kcl.mdeo.
uk.ac.kcl.mdeoptimiser

I would really want to use "uk.ac.kcl.mdeo." as it's simple and clear. Probably there should be a cleanup task to make these consistent everywhere.

Reply:
I have no issue with that :-)

Reply:
Great!